src/globalstate.h

```diff
@@ -19,8 +19,32 @@
 
 #define MAXPRINTDEPTH 1024L
 
+#if defined(HPCGAP)
+#define TLS_NUM_EXTRA 256
+#endif
+
 typedef struct GlobalState
 {
+#if defined(HPCGAP)
+  int threadID;
+  void *threadLock;
+  void *threadSignal;
+  void *acquiredMonitor;
+  unsigned multiplexRandomSeed;
+  void *currentRegion;
+  void *threadRegion;
+  void *traversalState;
+  Obj threadObject;
+  Obj tlRecords;
+  Obj lockStack;
+  int lockStackPointer;
+  Obj copiedObjs;
+  Obj interruptHandlers;
+  void *CurrentHashLock;
+  char *CurrFuncName;
+  int DisableGuards;
+#endif
+
```

Comment:
This sync of the two struts seems incomplete, as there are many more diffs between the two. Is this intentional?